roles/tas-install/charts/intel-telemetry-aware-scheduler/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```

Comment:
Nit: Should this say something about TAS ? I'm not sure where this description is picked up